frontend/my-app/src/AdminDesktop.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {useNavigate} from "react-router-dom";
+
+export const AdminDesktop = () => {
+
+    const navigate = useNavigate()
+    const addNewProductFunction = () => {
+        navigate('/AddNewProduct')
+    }
+
+    const checkProductsFunction = () => {
+        navigate('/checkOrders')
+    }
+
+    const returnToMainPage = () =>{
+        navigate('/')
+    }
+
+
+    const checkOrdersFunction = () => {

```

Comment:
It seems strange to me. Why is checking order named /orders and checking products named /checkOrders ?
This one should be /orders, and one above should be /products probably.